Mathlib/RepresentationTheory/FdRep.lean

```diff
@@ -197,3 +198,45 @@ theorem dualTensorIsoLinHom_hom_hom : (dualTensorIsoLinHom ρV W).hom.hom = dual
 #align fdRep.dual_tensor_iso_lin_hom_hom_hom FdRep.dualTensorIsoLinHom_hom_hom
 
 end FdRep
+
+noncomputable section equivFullSubcategory
+
+namespace FdRep
+
+variable {k G : Type u} [Field k] [Monoid G] {V W : FdRep k G}
+
+/- TODO: Unify how to phrase lemmas/theorems - Use V? V.toRep? Or V.ρ.asModule? -/
+instance toRep_finiteDimensional : FiniteDimensional k V.toRep :=
+  FGModuleCat.instFiniteCarrier k _
+
+/-- Lift `FdRep` morphisms to `Rep` morphisms via the forgetful functor. -/
+def lift_hom (f : V ⟶ W) : V.toRep ⟶ W.toRep :=
+  (forget₂ _ _).map f
+
+/-- Bundles Rep with a FiniteDimensional into a FdRep. -/
+def ofRep (V : Rep k G) [hV : FiniteDimensional k V] : FdRep k G :=
+  ⟨⟨V.V, hV⟩, V.ρ⟩
+
+/-- Functor from `FdRep` to the full subcategory of finite dimensional `Rep`. Also see
+`equivalenceFiniteDimensionalSubcategory` for an equivalence of category. -/
+def toFiniteDimensionalSubcategory :
+    FdRep k G ⥤ FullSubcategory (fun V : Rep k G ↦ FiniteDimensional k V) :=
+  FullSubcategory.lift _ (forget₂ _ _) inferInstance
+
+/-- Functor from the full subcategory of finite dimensional `Rep` to `FdRep`. Also see
+`equivalenceFiniteDimensionalSubcategory` for an equivalence of category. -/
+def ofFiniteDimensionalSubcategory :
+    FullSubcategory (fun V : Rep k G ↦ FiniteDimensional k V) ⥤ FdRep k G where
+  obj := fun ⟨V, _⟩ ↦ FdRep.ofRep V
+  map := fun f ↦ ⟨f.hom, f.comm⟩
+
+/-- Equivalence between `FdRep` and the full subcategory of finite dimensional `Rep`. -/
+def equivalenceFiniteDimensionalSubcategory :
+    FdRep k G ≌ FullSubcategory (fun V : Rep k G ↦ FiniteDimensional k V) where
+  functor := toFiniteDimensionalSubcategory
+  inverse := ofFiniteDimensionalSubcategory
+  unitIso := by aesop_cat
+  counitIso := by aesop_cat
```

Comment:
We should not construct data using such tactics. Did you try `unitIso := Iso.refl _`?